crates/iceberg/src/scan.rs

```diff
@@ -394,6 +411,11 @@ impl TableScan {
         return Ok(file_scan_task_rx.boxed());
     }
 
+    /// Returns an [`ManifestList`]
+    pub async fn get_manifest_list(&self) -> Result<Arc<ManifestList>> {
```

Comment:
It could be used by the query engine (datafusion/databend), we need to know the scan stats of this table (rows, bytes).

Reply:
> It could be used by the query engine (datafusion/databend), we need to know the scan stats of this table (rows, bytes).

Hi, seems this change not related to this PR. How about moving to another new PR?